HW_4/task_4.py

```diff
@@ -0,0 +1,125 @@
+import os
+from pathlib import Path
+import requests
+import time
+import argparse
+import threading
+from multiprocessing import Process
+import asyncio
+import aiohttp
+
+images = []
+with open('images.txt', 'r') as f:
+    for image in f.readlines():
+        images.append(image.strip())
```

Comment:
Но можно вообще иначе
```suggestion
with open(r'HW_4/images.txt') as f:
    images = f.read().split('\n')
```